### DISCUSSION TITLE
How to migrate my cmder configuration with zsh to laragon cmder

### DISCUSSION THREAD

**Discussion by alwiismail66** [leokhoa/laragon#773] [Q&A]
i have cmder with zsh and other configuration like color scheme and shortcut key but the problem is when i change the environment like change version php then php version won't change in my terminal so i try export conemu.xml then i replace conemu.xml in laragon directory but the problem same when environment change the terminal cannot detect it always detect environment before change how to resolve it

**Comment by carlitorweb** [comment]
You need change your path. Got to Tools -> Path -> Remove Laragon from Path. Then, same location and use Add Laragon to Path. Close your terminal and open again and `php -v` should be okay now

**Reply by alwiismail66** [reply]
still not working,btw i use the other task like bash in cmder ,but when i change the task in cmder using cmder is work properly without any change,why the cmder read path from laragon but other task don't

**Reply by alwiismail66** [reply]
your solution is correct but how to make automatic detect path from laragon like cmder task because the other task just read path from user setting environment variable not from laragon

**Reply by carlitorweb** [reply]
You need define in each bash the path to laragon stuff. ANd you need do it with each bash you will use. I think still better use lobal path from OS, where you define it once, with a couple of clicks
